ArcGISRuntimeSDKQt_CppSamples/Geometry/FormatCoordinates/FormatCoordinates.qml

```diff
@@ -0,0 +1,173 @@
+// [WriteFile Name=FormatCoordinates, Category=Geometry]
+// [Legal]
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import Esri.Samples 1.0
+import Esri.ArcGISExtras 1.1
+
+FormatCoordinatesSample {
+    id: formatCoordinates
+    clip: true
+    width: 800
+    height: 600
+
+    property double scaleFactor: System.displayScaleFactor
+    property int labelWidth: 200 * scaleFactor
+    property int coordinateTextWidth: 200 * scaleFactor
+    property int fontPixelSize: 14 * scaleFactor
+    property int textPadding: 4 * scaleFactor
+
+    MapView {
+        anchors {
+            top: parent.top
+            left: parent.left
+            right: parent.right
+            bottom: leftColumnRect.top
+        }
+
+        objectName: "mapView"
+    }
+
+    Rectangle {
```

Comment:
Optional, but a Column might be better suited than a Rectangle in this scenario. You could also consider a Repeater